sw-tests.js

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Yep, by now because I don't like at all you can not configure which file is. Indeed, this should lie on the `karma.conf.js` file.